test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -11093,21 +11103,19 @@ def add_server_dyn_res(self, custom_resource, script_body=None,
         :type suffix: str
         """
         if res_file is not None:
-            f = open(file)
-            script_body = f.readlines()
-            f.close()
-            self.du.chmod(self.hostname, path=res_file, mode=perm, sudo=True)
+            with open(res_file) as f:
+                script_body = f.readlines()
+                self.du.chmod(self.hostname, path=res_file, mode=perm,
```

Comment:
I don't think this chmod() has to be in the with statement.  The with statement should be the code that deals with the open file.  This happened after the close before, so shouldn't it happen after the with statement now?